modules/sandboxed-containers-metrics-list.adoc

```diff
@@ -0,0 +1,16 @@
+// Module is included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-metrics-list_{context}"]
+= About {sandboxed-containers-first} metrics
+
+{sandboxed-containers-first} metrics enable administrators to monitor how their sandboxed containers are running.
+
+{sandboxed-containers-first} metrics are collected for the following categories:
+
+Kata agent metrics:: Kata agent metrics display information about the kata agent process running in your sandboxed containers. These metrics include data from `/proc/<pid>/[io, stat, status]`.
+Kata guest OS metrics:: Kata guest OS metrics display data from the gues OS running in your sandboxed containers. These metrics include data from `/proc/[stats, diskstats, meminfo, vmstats]` and `/proc/net/dev`.
+Hypervisor metrics:: Hypervisor metrics display data regarding the hypervisor process running the sandboxed containers. These metrics mainly include data from `/proc/<pid>/[io, stat, status]`.
```

Comment:
Same level of detail: the hypervisor is running the VM, not the sandboxed containers.

Reply:
Do we want to mention which hypervisor. Also, is that the hypervisor or the VMM (KVM or QEMU?)  

cc @littlejawa

Reply:
@zanetworker - sorry I didn't see your comment before.
That would be QEMU. I don't know if we want to mention it, probably depends how we refer to it in other parts of the doc. But I don't see an issue with that.